helloworld.cpp

```diff
@@ -3,4 +3,5 @@
 Using namespase std;
```

Comment:
It should be "using namespace std;" (lower case "u", "namespace" instead of "namespase").